### PR TITLE
chore(serverless): add myself as owner

### PR DESCRIPTION
**Pull request by G-Rath**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

------

Adding myself as an owner as I'm happy to help maintain these packages, and this way it should be quicker to add and modify properties as needed (i.e I've got add a `patterns` property, and would prefer not to have to wait another month 😅)

### CODE REVIEW

File: types/serverless/index.d.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 //                 Alex Pavlenko <https://github.com/a-pavlenko>
 //                 Frédéric Barthelet <https://github.com/fredericbarthelet>
 //                 Bryan Hunter <https://github.com/bryan-hunter>
-//                 Thomas Aribart <https://github.com/thomasaribart>
```
Comment (peterblazejewicz) [inline types/serverless/index.d.ts:8]: You're dropping this GH handle, why?

Comment (G-Rath) [reply types/serverless/index.d.ts:8]: Because apparently I can't copy and paste 🤦

Comment (G-Rath) [reply types/serverless/index.d.ts:8]: Have restored this GH handle :)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
(way to go!)

**Comment by G-Rath** [comment]
@sandersn any chance of getting a quick DT maintainer approve?

**Pull request merged**